static/app/components/onboardingWizard/onboardingCard.tsx

```diff
@@ -0,0 +1,137 @@
+import React, {Fragment} from 'react';
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+import {PlatformIcon} from 'platformicons';
+
+import Card from 'sentry/components/card';
+import Link from 'sentry/components/links/link';
+import {IconChevron, IconEllipsis} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import pulsingIndicatorStyles from 'sentry/styles/pulsingIndicator';
+import space from 'sentry/styles/space';
+import {Organization, Project} from 'sentry/types';
+import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
+import withProjects from 'sentry/utils/withProjects';
+import {usePersistedOnboardingState} from 'sentry/views/onboarding/targetedOnboarding/types';
+
+const MAX_PROJECT_COUNT = 3;
+
+function OnboardingViewTask({
+  org,
+  projects: allProjects,
+}: {
+  org: Organization;
+  projects: Project[];
+}) {
+  if (!org?.experiments.TargetedOnboardingMultiSelectExperiment) {
+    return null;
+  }
+
+  const [onboardingState] = usePersistedOnboardingState();
+  if (!onboardingState) {
+    return null;
+  }
+  // Projects selected during onboarding but not received first event
+  const projects = onboardingState.selectedPlatforms
+    .map(platform => onboardingState.platformToProjectIdMap[platform])
```

Comment:
What's the reason for needing this `platformToProjectIdMap` state? Is this so we only have projects listed that were actually selected in onboarding? When does this state get cleaned up on the backend?

Reply:
This records the project ids of the projects that we created during onboarding. This is so that if the user goes back and do the multiplatform select step again, we won't create duplicated projects.

So yeah filtering allProjects based on this ensures that we only list projects created during onboarding.

This state is stored in redis on the backend and gets expired in 30 days